crates/fuel-core/src/service/query.rs

```diff
@@ -89,7 +89,7 @@ impl FuelService {
             db,
             tx_status_manager,
         };
-        Ok(transaction_status_change(state, rx, id).await)
+        Ok(transaction_status_change(state, rx, id, false).await)
```

Comment:
I think this function can also benefit from making it configurable.

But I'm fine to not propagate it=) But then I think it is better to use `true`, because we want to allow pre confirmations